redis/src/main/scala/zio/redis/api/Sets.scala

```diff
@@ -143,16 +141,17 @@ trait Sets {
    *
    * @param key Key of the set to scan
    * @param cursor Cursor to use for this iteration of scan
-   * @param regex Glob-style pattern that filters which elements are returned
+   * @param pattern Glob-style pattern that filters which elements are returned
    * @param count Count of elements. Roughly this number will be returned by Redis if possible
    * @return Returns the next cursor, and items for this iteration or nothing when you reach the end, as a tuple
    */
   final def sScan(
     key: String,
     cursor: Long,
-    regex: Option[Regex] = None,
-    count: Option[Count] = None
-  ): ZIO[RedisExecutor, RedisError, (Long, Chunk[String])] = SScan.run((key, cursor, regex, count))
+    pattern: Option[String] = None,
+    count: Option[Long] = None
```

Comment:
I think it's fine to have `Count` here (propagate to all others as it's not in sync).

Reply:
Changed back to Count